x-pack/plugins/apm/public/components/routing/templates/apm_service_template/index.tsx

```diff
@@ -220,6 +221,16 @@ function useTabs({ selectedTab }: { selectedTab: Tab['key'] }) {
         defaultMessage: 'Service Map',
       }),
     },
+    {
+      key: 'logs',
+      href: router.link('/services/:serviceName/logs', {
+        path: { serviceName },
+        query,
+      }),
+      label: i18n.translate('xpack.apm.home.serviceLogsTabLabel', {
+        defaultMessage: 'Logs',
+      }),
```

Comment:
To @alex-fedotyev point about hiding for RUM

```suggestion
      label: i18n.translate('xpack.apm.home.serviceLogsTabLabel', {
        defaultMessage: 'Logs',
      }),
      hidden: !agentName || isRumAgentName(agentName) || isIosAgentName(agentName)
```